### ISSUE TITLE
Firefox version

### ISSUE THREAD

**Issue by dmose** [oblador/loki#141]
It would be wonderful if this worked with Firefox as well.  This seems to be mentioned in some asides in #71, but without really being addressed there in detail.

**Comment by stefankendall** [comment]
100%. This would be a great improvement.

**Comment by elchiconube** [comment]
I agree, with Firefox we can cover a great % for testing :) Pliiiiiiiis
